webapps-backup/src/main/java/io/camunda/webapps/backup/repository/opensearch/OpenSearchGetSnapshotResponse.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.backup.repository.opensearch;
+
+import java.util.List;
+import java.util.Objects;
+import org.opensearch.client.opensearch.snapshot.GetSnapshotResponse;
+
+public class OpenSearchGetSnapshotResponse {
```

Comment:
OpenSearchGetSnapshotResponse from operate.
The class in operate is still in use in `OpenSearchSnapshotOperations` which are not used atm, in the future we may remove them from operate as well